client/src/environments/environment.dev.ts

```diff
@@ -0,0 +1,3 @@
+export const environment = {
+  apiUrl: 'https://dev-vocabtrainer-api.taskbase.com/api',
```

Comment:
we have twice `api` in the url. Maybe rename `api` in the subdomain to `backend` or we get rid of the `/api` path prefix. What do you say?

Reply:
i wanted to keep the `/api` in case we host it together to avoid collisions. so let's go with the `-backend` then.